src/02_datatypes.py

```diff
@@ -9,13 +9,16 @@
 """
 
 x = 5
-y = "7"
+y = 7
 
 # Write a print statement that combines x + y into the integer value 12
 
 # YOUR CODE HERE
-
+print(x+y)
 
 # Write a print statement that combines x + y into the string value 57
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
```

Comment:
There is a process called typecasting, that allows you to change one type of variable such as string into an integer.